### ISSUE TITLE
Colorado county routes (Minor roads with ref, but no name?)

### ISSUE THREAD

**Issue by bradrh1** [osm-americana/openstreetmap-americana#564]
I noticed that, at least in my area, county roads are not labeled?   Also USFS & BLM roads have no label.   For example:  https://www.openstreetmap.org/way/245698608
It looks like only names are rendered, not refs?

Tasks (edit by @ZeLonewolf)
* [ ] Map routes to `network=US:CO:(whatever)` route relations
* [ ] Ensure network value is current supported, or add support in the style

**Comment by ZeLonewolf** [comment]
That's right - in general, route numbers go on route relations and names get put on the way, so we'll render a shield for a route and a name on the road itself.  This is the behavior in OpenMapTiles.

**Comment by bradrh1** [comment]
So, the standard layer on openstreepmap.org does not use OpenMapTiles?   I ask because they do show name labels derived from refs on roads which are not part of a route relation

**Comment by ZeLonewolf** [comment]
> So, the standard layer on openstreepmap.org does not use OpenMapTiles? I ask because they do show name labels derived from refs on roads which are not part of a route relation

Correct.  The standard tile layer ignores route relations entirely.

**Comment by ZeLonewolf** [comment]
A discussion on this topic is ongoing on Slack:
https://osmus.slack.com/archives/C7QHYF47L/p1667231962869639

**Comment by ZeLonewolf** [comment]
On discussion, it appears that the cited road is a Colorado county route, with the MUTCD-standard blue pentagon shield displayed on the ground to indicate that it's a route.  Therefore, the road should be added to an appropriate route relation.  If Colorado county routes aren't defined yet in the style, that will need to get done as well.

**Comment by claysmalley** [comment]
To see a list of Colorado county road networks supported by Americana, check the [Taginfo project page](https://taginfo.openstreetmap.org/projects/osm_americana#tags) and filter the list for `US:CO:`.

As of writing, there is one relation with `network=US:CO:Douglas` and one relation with `network=US:CO:Larimer`. There are also 25 ways with `network=US:CO:Chaffee`, but none of them belong to relations.
